.env

```diff
@@ -1,4 +1,3 @@
-NODE_PATH=src
-PORT=8080
+PORT=3000
```

Comment:
I believe this is a regression. The port should be 8080 so it doesn't interfere with rails (if someone wants to run it).